src/less/components/breadcrumb/index.less

```diff
@@ -1,17 +1,22 @@
 /* ========== hxBreadcrumbs ========== */
 .hxBreadcrumb {
+  display: flex;
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
   align-content: center;
   align-items: center;
-  display: flex;
+  line-height: 1rem;
   font-size: 0.75rem;
   font-weight: 500;
-  height: 1.75rem;
-  margin-bottom: 1rem;
-  text-transform: uppercase;
+  text-transform: lowercase;
 
   & > a {
     margin-right: 0.5rem;
 
+    &::first-letter {
+      text-transform: uppercase;
+    }
```

Comment:
This won't apply proper casing for multi-word links. A link with text `foo bar` would become `Foo bar`, when it should be `Foo Bar`.

This can be improved by using [`text-transform: capitalize;`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform#capitalize_(General)) instead.

```less
& > a {
  text-transform: capitalize;

  /* ... */
}
```

Reply:
Done.